ci/build_container/build_recipes/luajit.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-VERSION=2.0.5
+VERSION=2.1.0-beta3
```

Comment:
I don't personally have an objection to upgrading to 2.1, but worth a discussion if anyone else cares about this.